packages/forms-web-app/__tests__/unit/routes/before-you-start/index.test.js

```diff
@@ -0,0 +1,18 @@
+const { use } = require('../router-mock');
+
+describe('routes/before-you-start/index', () => {
+  beforeEach(() => {
+    jest.resetModules();
+
+    // eslint-disable-next-line global-require
+    require('../../../../src/routes/before-you-start');
+  });
+
+  afterEach(() => {
```

Comment:
We don't need afterEach as there is only one test exists

Reply:
Removed

Reply:
Leaving this test as it is to keep consistency with other sections.